### ISSUE TITLE
Windows Authentication Dialog issues

### ISSUE THREAD

**Issue by stormcoder** [codeborne/selenide#432]
For our pre-integration environment, we need to use a Windows Authentication dialog with IE 11. We use an AutoIt script for interacting with the dialog. While selenide is running, the dialog can't be interacted with either manually or with the script. The active control is always the password field no matter what we do. If I manually click into the username field, the cursor immediately drops into the password field. So when our script runs, all the text is sent to the password field. As soon as Selenide stops running (I kill selenide and IE stays up), the dialog immediately starts working properly and we can use our script on that dialog without a problem or manually operate on the dialog. I have run the tests in straight Selenium and the issue with the dialog doesn't occur. I have tried using the domain authentication in the open method and this doesn't work at all and the problem with the authentication dialog still occurs. I'm at a loss on why this is happening. 

Selenide has solved a lot of problems for us but this problem with the Windows Authentication dialog is a show stopper and I have to go back to just plain selenium which is unfortunate.

**Comment by asolntsev** [comment]
@stormcoder How can I reproduce the issue? Without being able to reproduce, I cannot fix the issue. 

NB! Why the hell do you need authentication on test environment? Just disable it. It's not the application functionality, but an infrastructure - so you don't need to check it with functional tests.

**Comment by iCok** [comment]
In addition to comment @asolntsev:
1) If it's base auth, you can try to put your username and password directly to url. 
(Example: http://user:password@www.stagesite.com)
2) Set-up your proxy and set username, password there.

**Comment by stormcoder** [comment]
@asolntsev To reproduce you need a web server setup for Domain Authentication. That should have been obvious when I said, "we need to use a Windows Authentication dialog with IE 11". Labeling this "Cannot Reproduce", is just lazy. You could reproduce it if you desired.

I have to use Domain Authentication because that is what is required by the company. I don't get a choice in this. It is the Selenide because this does not happen when I just use Selenium. Saying it's not a bug because you don't have to use Domain Authentication is bullshit. I do not need Selenide to do the authentication. If you actually read my report, you will see that I said that Selenide grabs the dialog and makes it so we can't enter a domain/username. Telling me that I should just not do something is not at all helpful.

**Comment by stormcoder** [comment]
@iCok Item one does not work because it is not Domain Authentication and that is what I am required to use and it does not involve the actual logon. The problem is that Selenide grabs the authentication dialog and won't allow use to enther a Domain\username. This does not happen with plain Selenium. Item two is just a restatement of item one.

**Comment by stormcoder** [comment]
I've had to replace the Selenide functionality in our framework. So don't fix the bug, if you don't want to. Doesn't matter to me. I will, however, be cautioning other people against making use of selenide.

**Comment by asolntsev** [comment]
@stormcoder Sorry for the long delay, I was in hurry with my work, and didn't have chance to reply to your previous comments in time. :(

Also I am sorry my initial answer. Apparently it may seemed to be rude and caused a negative reaction from you. 

Please understand that I am asking "How can I reproduce the issue?" not because I am lazy. It's because I don't have time to reproduce everyone's issues. I don't have time to configure web servers for Domain Authentication. I don't have any web servers to configure. 

> "Open source" doesn't mean that somebody will fix your issues for free. "Open source" means that you are free to fix the issue and send a pull request. And as a bonus, sometimes somebody can fix the issue for you. 

Let me rephrase my question: **How can you help me to reproduce the issue?**
I will fix the issue for you with a great pleasure, but _I need your help_.

**Comment by rosolko** [comment]
If needed reopen with additional info.
